hw/xbox/nv2a/swizzle.c

```diff
@@ -36,21 +36,33 @@
 static void generate_swizzle_masks(unsigned int width,
                                    unsigned int height,
                                    unsigned int depth,
-                                   uint32_t* mask_x,
-                                   uint32_t* mask_y,
-                                   uint32_t* mask_z)
+                                   uint32_t *mask_x,
+                                   uint32_t *mask_y,
+                                   uint32_t *mask_z)
 {
     uint32_t x = 0, y = 0, z = 0;
     uint32_t bit = 1;
     uint32_t mask_bit = 1;
     bool done;
     do {
         done = true;
-        if (bit < width) { x |= mask_bit; mask_bit <<= 1; done = false; }
-        if (bit < height) { y |= mask_bit; mask_bit <<= 1; done = false; }
-        if (bit < depth) { z |= mask_bit; mask_bit <<= 1; done = false; }
+        if (bit < width) {
+            x |= mask_bit;
+            mask_bit <<= 1;
+            done = false;
+        }
+        if (bit < height) {
+            y |= mask_bit;
+            mask_bit <<= 1;
+            done = false;
+        }
+        if (bit < depth) {
+            z |= mask_bit;
+            mask_bit <<= 1;
+            done = false;
+        }
```

Comment:
Harder to read now in my opinion.

Reply:
I think it's fine either way, we can revert this change